.travis.yml

```diff
@@ -1,5 +1,21 @@
 language: node_js
 node_js:
     - "8"
-script: ./build/ci.sh
-cache: yarn
+cache:
+  yarn: true
+  directories:
+    - node_modules
+jobs:
+  include:
+    - # commit lint
+      script: commitlint-travis
+    - # code lint
+      script: npm-run-all clean build:i18n lint
+    - # flow types
+      script: npm-run-all clean build:i18n; yarn flow
```

Comment:
Just checking, is the `;` before `yarn flow` intentional? If so, what does it do and why here but not in the other `npm-run-all` scripts?

Reply:
; is same as && for shell